Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Undo/PrefabUndo.cpp

```diff
@@ -104,23 +88,15 @@ namespace AzToolsFramework
             m_entityAlias = aliasReference.value();
 
             //generate undo/redo patches
-            m_instanceToTemplateInterface->GeneratePatch(m_redoPatch, initialState, endState);
-            m_instanceToTemplateInterface->AppendEntityAliasToPatchPaths(m_redoPatch, entityId);
-            
-            m_instanceToTemplateInterface->GeneratePatch(m_undoPatch, endState, initialState);
-            m_instanceToTemplateInterface->AppendEntityAliasToPatchPaths(m_undoPatch, entityId);
+            const AZStd::string& entityAliasPath = m_instanceToTemplateInterface->GenerateEntityAliasPath(entityId);
+            PrefabUndoUtils::GenerateUpdateEntityPatch(m_redoPatch, initialState, endState, entityAliasPath);
+            PrefabUndoUtils::GenerateUpdateEntityPatch(m_redoPatch, endState, initialState, entityAliasPath);
```

Comment:
Should this be m_undoPatch?

Reply:
Nice catch! I have a local commit that I forgot to push.